source/ImageSet.h

```diff
@@ -46,7 +46,7 @@ class ImageSet {
 public:
 	// ImageSets should be created with a name, as some image paths (e.g. plugin icons)
 	// do not contain the associated image name.
-	ImageSet(std::string name);
+	explicit ImageSet(std::string name);
```

Comment:
const &?

Reply:
I think you get the same number of copies if the constructor uses std::move, but I'm not sure...

Reply:
```suggestion
	explicit ImageSet(const std::string &name);
```